Daniel-L-Ross/kennels
# Dr creating kennel component

# Description

Chapteres 1-3 of book4 content. Creating my first react app, following the NSS content.

Fixes # (1)

## Type of change


# Testing Instructions

git fetch --all
git checkout dr-creating-kennel-component

dom should render with dogs, employees, customer, and locations. The button should increase the counter above the button. 


# Checklist:
